ddl/constant.go

```diff
@@ -25,10 +25,10 @@ const (
 	ReorgTable = "tidb_ddl_reorg"
 	// HistoryTable stores the history DDL jobs.
 	HistoryTable = "tidb_ddl_history"
-	// BackfillTable stores the information of backfill jobs.
-	BackfillTable = "tidb_ddl_backfill"
-	// BackfillHistoryTable stores the information of history backfill jobs.
-	BackfillHistoryTable = "tidb_ddl_backfill_history"
+	// BackgroundSubtasksTable stores the information of background jobs.
+	BackgroundSubtasksTable = "tidb_background_subtasks"
```

Comment:
```suggestion
	BackgroundSubtaskTable = "tidb_background_subtask"
```

We needn't `s`.
e.g.  `mysql.user` not `mysql.users`